pkg/client/client.go

```diff
@@ -416,3 +442,27 @@ func (c *Client) indexes(tableName string) ([][]string, []string, error) {
 
 	return c.Query(query, args...)
 }
+
+func getDB(driver, connString, database string) (*sqlx.DB, error) {
+	var newConnString string
+
+	if driver == drivers.MySQL {
+		newConnString = strings.Replace(connString, "/", fmt.Sprintf("/%s", database), 1)
+	} else {
+		u, err := url.Parse(connString)
+		if err != nil {
+			return nil, err
+		}
+
+		u.Path = "/" + database
+		newConnString = u.String()
+	}
```

Comment:
I would use a switch and return an error when it's none of the supported database types

I'm a bit worried by the fact you will apply the postgres way to any future database type that might be added

Reply:
I agree with using a switch here, but the driver validation is done somewhere. If the execution reaches this point, the driver is actually valid. Regarding applying the postgres way, yeah a switch statement might help supporting a new use case.